wordpress-mobile/WordPress-Android
# Fix pages menu

Fixes #8520.

To test:
1. Go to ***My Site*** tab.
2. Tap ***Site Pages*** item under ***Publish*** section.
3. Notice overflow menu is shown for each page list item.
4. Tap page list item.
5. Notice editor is opened.
6. Tap back arrow.
7. Notice overflow menu is shown for each page list item.
8. Swipe down to refresh list.
9. Notice overflow menu is still visible
